junit5/src/main/java/com/github/automatedowl/tools/JSErrorsExtension.java

```diff
@@ -49,14 +49,14 @@ public void afterTestExecution(ExtensionContext context) {
 
         // Check for JS errors assertion boolean flag.
         // Skip error throwing in case of negative unit test.
-        if (isAssertJSErrorsEnabled(context) && getJSErrorsFromLogEntries(logEntries).anyMatch(e -> true)
+        if (isAssertJSErrorsEnabled(context) && getJSErrorsFromLogEntries(logEntries).count()!=0
                 && context.getTestClass().toString().contains("com.github.automatedowl")
                 && context.getRequiredTestMethod().getName().equals("referenceErrorTest")) {
             assertThrows(WebDriverException.class, ()->{
                 throw new WebDriverException(JS_ERRORS_EXCEPTION_STRING);
             });
-        } else if (isAssertJSErrorsEnabled(context) && getJSErrorsFromLogEntries(logEntries).anyMatch(e -> true)) {
-            throw new WebDriverException(JS_ERRORS_EXCEPTION_STRING);
+        } else if (isAssertJSErrorsEnabled(context) && getJSErrorsFromLogEntries(logEntries).count()!=0) {
```

Comment:
What's the purpose of the change from anyMatch to count? @mkulikov

Reply:
Logic above returns true when there is items in list, which is more confusing than simple check for count != 0